Sources/App/Core/Search.swift

```diff
@@ -105,6 +105,7 @@ enum Search {
             .where(isNotNull(repoName))
             .orderBy(eq(lower(packageName), mergedTerms), .descending)
             .orderBy(score, SQLDirection.descending)
+            .orderBy(packageName, SQLDirection.ascending)
```

Comment:
The order can change slightly between runs, because ordering by score is not very granular (there are many packages which share a score). Pagination makes that more obvious when we slice into the set and results end up on different pages between requests.

Reply:
Yep, this is perfect 👍